rai/client.go

```diff
@@ -342,7 +354,7 @@ func readArrowFiles(files []TransactionAsyncFile) ([]ArrowRelation, error) {
 					data, _ := rec.Column(i).MarshalJSON()
 					var values []interface{}
 					json.Unmarshal(data, &values)
-					out = append(out, ArrowRelation{rec.ColumnName(i), values})
+					out = append(out, ArrowRelation{file.Name, values})
```

Comment:
This is unrelated to the current PR, but I'm wondering what the `json.Unmarshal` is doing here in `readArrowFiles`? :) 
The change is unrelated as well, right?

Reply:
the `data, _ := rec.Column(i).MarshalJSON()` returns a `byte array` one way to convert it to an array of values is to use 
```
var values []interface{}
json.Unmarshall(data, &values)
```
Yep the change is unrelated and it's mainly added to get aligned with `js-sdk` it sets the relation id to the rel path.